activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-core/src/main/java/org/activiti/cloud/services/core/ProcessVariableJsonNodeConverter.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.cloud.services.core;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class ProcessVariableJsonNodeConverter implements ProcessVariableValueSpringConverter<JsonNode> {
+
+    private final ObjectMapper objectMapper;
+
+    public ProcessVariableJsonNodeConverter() {
+        this.objectMapper = new ObjectMapper();
```

Comment:
It's probably better to inject ObjectMapper instead of instantiating a new one. So it can take benefit from potential customizations.